ci.nix

```diff
@@ -69,6 +69,9 @@ let
           # build the shell expression to be sure it works on all platforms
           shell = import ./shell.nix { inherit packages; };
 
+          # build deployment tools
+          inherit (pkgs) thorp;
```

Comment:
```suggestion
          inherit (plutus) thorp;
```

Reply:
just worked it out before I saw your comment 🥇